desktop/core/src/desktop/templates/scheduler/submit_job_popup.mako

```diff
@@ -128,6 +128,10 @@ else:
   }
   #param-container .btn-group {
     margin-left: 10px;
+    position: fixed;
+  }
+  #param-container .btn-group.open {
+    position: absolute !important;
```

Comment:
Do you need !important here? If so perhaps some other class is more specific...

Reply:
I've removed this **#param-container .btn-group.open** style. 
Earlier to acknowledge the issue where the calendar button moved from its place when we zoomed in, I added the position as fixed. Because of this fixed position, when we opened the calendar button it overlapped with the button below that (as shown in the attached image), hence I used the !important with position style of **#param-container .btn-group.open**. 

The ideal solution would be to have an absolute position for the **#param-container .btn-group** style. 

<img width="608" alt="Screenshot 2024-06-13 at 10 18 16 PM" src="https://github.com/cloudera/hue/assets/21025848/26b8d232-0921-4f8b-852a-c5ee9cf877a9">